.github/workflows/continous-integration.yml

```diff
@@ -228,12 +228,11 @@ jobs:
         poetry config virtualenvs.in-project true
 
     - name: Load Poetry Cached Libraries ⬇
-      if: needs.changes.outputs.backend == 'true'
+      if: needs.changes.outputs.backend == 'true' && matrix.os != 'windows-latest'
       uses: actions/cache@v1
       with:
         path: .venv
-        key: ${{ runner.os }}-poetry-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
-        restore-keys: ${{ runner.os }}-poetry-${{ matrix.python-version }}
+        key: ${{ runner.os }}-poetry-${{ env.POETRY_VERSION }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
```

Comment:
do we want to include the `POETRY_VERSION` for the cache key of other workflows?

Reply:
Hmm, yes! Thanks for pointing it out.